docs/source/models/gbd2017/causes/cause_model_template.rst

```diff
@@ -214,6 +214,66 @@ Validation Criteria
 References
 ----------
 
+.. [GBD-2017-YLD-Appendix-Cause-Model-Template]
+
+   Pages ???-??? in `Supplementary appendex 1 to the GBD 2017 YLD Capstone <YLD
```

Comment:
Realistically i'm just going to command-F the document, so I would shy away from adding page numbers---but I'm open to it.